sim/build.sbt

```
@@ -8,12 +8,9 @@ lazy val commonSettings = Seq(
   organization := "berkeley",
   version      := "1.0",
   scalaVersion := "2.12.10",
-  traceLevel   := 15,
   scalacOptions ++= Seq("-deprecation","-unchecked","-Xsource:2.11"),
-  libraryDependencies += "org.scalatest" %% "scalatest" % "3.1.0" % "test",
-  // Remove after bump to FIRRTL 1.3
-  libraryDependencies += "org.scalatestplus" %% "scalacheck-1-14" % "3.1.0.1" % "test",
-  libraryDependencies += "org.json4s" %% "json4s-native" % "3.6.1",
+  libraryDependencies += "org.scalatest" %% "scalatest" % "3.2.2" % "test",
+  libraryDependencies += "org.json4s" %% "json4s-native" % "3.6.10",
   libraryDependencies += "org.scala-lang" % "scala-reflect" % scalaVersion.value,
   addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full),
```

```suggestion
  addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.1" cross CrossVersion.full),
```
The recent bump of scalaVersion to 2.12.12 for mdf causes me to hit:
```
[error] stack trace is suppressed; run last mdf / update for the full output
[error] (mdf / update) sbt.librarymanagement.ResolveException: Error downloading org.scalamacros:paradise_2.12.12:2.1.0
[error]   Not found
[error]   Not found
[error]   not found: /home/centos/.ivy2/local/org.scalamacros/paradise_2.12.12/2.1.0/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/org/scalamacros/paradise_2.12.12/2.1.0/paradise_2.12.12-2.1.0.pom
[error]   not found: https://oss.sonatype.org/content/repositories/snapshots/org/scalamacros/paradise_2.12.12/2.1.0/paradise_2.12.12-2.1.0.pom
[error]   not found: https://oss.sonatype.org/content/repositories/releases/org/scalamacros/paradise_2.12.12/2.1.0/paradise_2.12.12-2.1.0.pom
[error]   not found: /home/centos/.m2/repository/org/scalamacros/paradise_2.12.12/2.1.0/paradise_2.12.12-2.1.0.pom
```

Bumping paradise to version 2.1.1 in CY fixes the issue.  We may want to bump to 2.1.1 here as well.